addons/project/static/src/views/project_task_form/project_task_form_controller.js

```diff
@@ -44,10 +44,13 @@ export class ProjectTaskFormController extends FormController {
             return;
         }
 
+        if (!record.resId) {
+             return;
+        }
```

Comment:
If it is not needed then we could remove it and we will see if the tour works. 🤷🏼‍♂️

Reply:
Done, i removed it